order/manager.go

```diff
@@ -308,6 +309,16 @@ func (m *Manager) BatchFinalize(batchID BatchID) error {
 	return nil
 }
 
+// OurNodePubkey returns our lnd node's public identity key or an error if the
+// manager wasn't fully started yet.
+func (m *Manager) OurNodePubkey() ([33]byte, error) {
+	if m.ourNodeInfo == nil {
```

Comment:
is this racy?

Reply:
Yes, good point. Added an atomic `isStarted` flag to the manager.